canonical-websites/insights.ubuntu.com
# Update the author pattern

## Done
Update the author pattern 

## QA
- Go to a post and check the author pattern is the round media object pattern.
- Check the author page has a round image too

## Details
Fixes https://github.com/canonical-websites/insights.ubuntu.com/issues/164 https://github.com/canonical-websites/insights.ubuntu.com/issues/167
